cmd/pinniped-proxy/Cargo.lock

```diff
@@ -1383,7 +1370,7 @@ dependencies = [
  "reqwest",
  "serde",
  "serde_json",
- "serial_test",
+ "temp-env",
```

Comment:
As we are waiting for the OSM team to provide the licenses file, we need to hold this change back for now. Please, do not merge it until the release 2.6.1 has been triggered 🙏

Reply:
Given that this is a `dev-dependency` only, I wonder if we can update the licenses file to exclude dev-only dependencies?